operations/network/network_management_install_guide/aruba/check_dhcp_lease_is_getting_allocated.md

```diff
@@ -0,0 +1,16 @@
+# Check DHCP lease is getting allocated
+
+* Check the KEA logs and verify that the lease is getting allocated.
+
+```
+kubectl logs -n services pod/$(kubectl get -n services pods | grep kea | head -n1 | cut -f 1 -d ' ') -c cray-dhcp-kea
+```
+
+2021-04-21 00:13:05.416 INFO  [kea-dhcp4.leases/24.139710796402304] DHCP4_LEASE_ ***ALLOC*** [hwtype=1 02:23:28:01:30:10], cid=[00:78:39:30:30:30:63:31:73:30:62:31], tid=0x21f2433a: lease 10.104.0.23 has been allocated for 300 seconds
```

Comment:
Should this be in code format?

Reply:
I was toying with both, the code format and just regular text. I could not bold the ALLOC message when in code format. I can change the portion into code format if it makes sense. I was just worried of people not seeing the ALLOC in the middle of logs.